static/css/restyle.less

```diff
@@ -723,28 +717,130 @@ body:not(.developer-hub) .secondary .highlight {
   padding: 7px 0 0;
 }
 
-// Install Button
-.button,
+// Button overrides.
+// Add a selector here if there's a different style coming from
+// elsewhere.
+#upload-file-widget .button.prominent,
+#promos .view-button a,
+.cta a.button:link,
 button,
-input[type="submit"] {
+.button,
+a.button:link,
+input:not(.upvotes):not(.downvotes)[type="submit"] {
   background: #5784bf;
-  border: 0;
+  border: none;
+  border-radius: 3px;
+  box-shadow: 0 2px 0 0 rgba(0, 0, 0, 0.10),
+        inset 0 -2px 0 0 rgba(0, 0, 0, 0.10);
+  color: #fff;
+  display: inline-block;
+  font-family: @primary-font;
+  font-size: 1.077rem;
+  line-height: 1.5;
+  padding: 8px 16px 9px;
+  position: relative;
+  text-shadow: none;
+  transition: box-shadow linear 0.25s;
+
+  &.download,
+  &.add {
+    background: #57bd35;
+    box-shadow: 0 2px 0 0 rgba(0, 0, 0, 0.10),
+        inset 0 -2px 0 0 rgba(56, 136, 30, 0.50);
+
+    span {
+      border: none;
+      margin: 0;
+      background: url(../img/impala/button-icons.png) no-repeat 0 -191px;
+      padding: 0 0 0 24px;
+    }
+  }
+
+  &.good {
+    background: #489615;
+  }
+
+  &.developer {
+    background: #bc2b1a;
+    span {
+      background-position: 0 -385px;
+    }
+  }
+
+  &.disabled,
+  &:disabled {
+    background: #ccc;
+    text-shadow: none;
+
+    &:hover {
+      box-shadow: 0 2px 0 0 rgba(0, 0, 0, 0.10),
+        inset 0 -2px 0 0 rgba(0, 0, 0, 0.10);
```

Comment:
This might be easier to read if the numbers lined up, so this indented a bit more. I believe it's done that way elsewhere in the patch, which is :+1: